src/index.js

```diff
@@ -1,15 +1,73 @@
-import 'bootstrap';
-import 'bootstrap/dist/css/bootstrap.min.css';
-import './css/style.css';
+import { renderLists, renderListContent, renderActiveList } from './components/renderDom';
+import {
+  displayAddBox, removeBox, displayNewListBox, confirmBoxLists, displayEditListBox,
+} from './components/renderPopUpBox';
+import {
+  addNewToDo, addNewList, setActiveList, toggleToDoStatus, removeList, removeToDo, editList,
+} from './components/toDoController';
 
-import basic from './components/basic'
+/* eslint-disable no-console */
+/* eslint-disable default-case */
 
-const basicPage = () => {
-    const parent = document.createElement('div');
-    parent.innerHTML = `
-          ${basic().outerHTML}
-      `;
-    return parent;
-};
+document.body.addEventListener('click', e => {
+  const target = e.target.className;
 
-document.getElementById('wrapper').appendChild(basicPage())
\ No newline at end of file
+  switch (target) {
+    case 'add-btn':
+      document.body.append(displayAddBox());
+      break;
+    case 'trash-svg-todos':
+      removeToDo(e.target.parentElement.parentElement);
+      break;
+    case 'remove-todo-btn':
+      removeToDo(e.target);
+      break;
+    case 'box-cancel-btn':
+      removeBox();
+      break;
+    case 'add-todo-btn':
+      addNewToDo();
+      renderListContent();
+      break;
+    case 'lists-add-btn':
+      document.body.appendChild(displayNewListBox());
+      break;
+    case 'trash-svg-lists':
+      document.body.appendChild(confirmBoxLists(e.target.parentElement.parentElement));
+      console.log(e.target.parentElement.parentElement);
```

Comment:
- [ ] Kindly remove this console log statement here. Debugging code shouldn't be committed.